docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4647,9 +4647,9 @@ Return low-level information on the node `id`
 ### Remove a node
 
 
-`DELETE /nodes/(id)`
+`DELETE /nodes/(id or name)`
 
-Remove a node [`id`] from the swarm.
+Remove a node `id` from the swarm.
```

Comment:
I'd remove `id` here, just "Remove a node from the swarm." that describes what the endpoint does